package.json

```diff
@@ -72,8 +72,10 @@
   "devDependencies": {
     "@types/jest": "24.9.1",
     "@types/jest-axe": "3.2.2",
+    "@types/node": "10.17.55",
     "@types/prettier": "1.19.1",
     "@types/webpack-dev-middleware": "4.1.0",
+    "@types/webpack-env": "1.16.0",
```

Comment:
**chore:** lets remove this completely  as it's not used anywhere

Reply:
It's used in a couple projects for webpack require() and require.context.

Reply:
thanks for info, that was not obvious from this PR :) too many changes in one bucket

Reply:
I mentioned this in the description: "Remove [webpack-env] references from tsconfigs except in the handful of packages that need it"